### PR TITLE
fix toogle button bug with theme

### PR DESCRIPTION
**Pull request by edwinfelipe**
My solution was to refactor the useThemeState function to make it call setValue on initialload without write the theme on local storage, if there is a theme variable on local storage use it else use the default value, and on the Toogle onChange function whrite the new value on local storage, the main problem was a bad use of the useEffect function.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,6 @@
     "gatsby-plugin-material-ui": "^2.1.10",
     "gatsby-plugin-react-helmet": "^3.3.13",
     "gatsby-plugin-sharp": "^2.6.39",
-    "gatsby-transformer-json": "^2.4.14",
```
Comment (iamdarshshah) [inline package.json:26]: Have already updated this. Remove this change.

**Review by iamdarshshah — CHANGES_REQUESTED** [review]
Have suggested removing the change in `package.json`, as it is already updated. Otherwise, this works as expected. Thanks @edwinfelipe 👍

File: package.json (hunk 1 above)
Comment (edwinfelipe) [reply package.json]: Sorry, now it is resolved

**Review by iamdarshshah — APPROVED** [review]

**Comment by iamdarshshah** [comment]
**Fixes** #97 🎉

**Pull request merged**